src/LibSingular.jl

```diff
@@ -1,11 +1,16 @@
 module libSingular
 
 import Libdl
+
+# We need the top-level eval for now
+import ..Singular
```

Comment:
Can you explain where you are using this. Normally I would say that this indicates that you have not got a clean separation between libsingular and Singular namespaces. But as you added a comment, I assume this was actually necessary.

Reply:
I think it is also a leftover and can be removed. The caller files are mostly removed from the `LibSingular` module anyway.